File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Display "Positive Number: " to prompt the user to input a positive integer from 1 to 8 and store it in n
    int n;
    do
    {
        n = get_int("Positive Number: ");
    }
    while (n < 1 || n > 8);

    // Iterate through each row
    for (int row = 0; row < n; row++)
    {
        // Print 'n - row' spaces
        for (int i = 0; i < n - row - 1; i++)
        {
            printf(" ");
        }

        // Print 'row' hash symbols ('#')
        for (int j = 0; j < row + 1; j++)
        {
            printf("#");
        }

        // Move to the next line
        printf("\n");
    }

    return 0;
}

File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


def main():
    # Prompt the user to input a positive integer between 1 and 8
    while True:
        n = get_int("Height: ")
        if 1 <= n <= 8:
            break

    # For each row from '1' to 'n'
    for row in range(1, n + 1):
        spaces = " " * (n - row)
        left_hashes = "#" * row

        # Print 'n - row' spaces and 'row' hash symbols
        print(f"{spaces}{left_hashes}")


if __name__ == "__main__":
    main()
